src/libm/sleefsimdsp.c

```diff
@@ -1712,6 +1752,27 @@ EXPORT CONST vfloat xfmodf(vfloat x, vfloat y) {
   return ret;
 }
 
+EXPORT CONST vfloat xsinpif_u05(vfloat d) {
```

Comment:
This and the following 4 functions does not look correct. Why do you have these dummy implementations?